content-services/content-sources-backend
# HMS-5125: Add support for resumable downloads

## Summary
Adds support for resumable downloads. 

Can test with [this](https://github.com/content-services/content-sources-frontend/pull/397) front-end PR.


## Testing steps

- Create an upoad repository.
- Select an item for upload, allow it to upload around halfway and then immediately hit refresh.
- After refresh, choose the same item, you should be able to see that item continue where you left off. 
- After this item has been completed, upload it, and create another upload repository. 
- Choose the same file, it should instantly be available.
